homeassistant/components/verisure/coordinator.py

```diff
@@ -101,8 +102,18 @@ async def _async_update_data(self) -> dict:
             "smart_plugs": {
                 device["deviceLabel"]: device for device in overview["smartPlugs"]
             },
+            "firmware": {
+                "latest": firmware_status["latestFirmware"],
+                "current": firmware_status["gateways"][0]["reportedRunningFirmware"],
+                "upgradeable": firmware_status["upgradeable"],
+            },
         }
 
+    @Throttle(timedelta(days=1))
+    async def _get_firmware_status(self) -> None:
+        """Get the firmware status."""
+        return await self.hass.async_add_executor_job(self.verisure.get_firmware_status)
```

Comment:
So, here is the thing I found out during testing:

```txt
2021-04-26 22:38:30 ERROR (MainThread) [homeassistant.components.verisure] Could not read overview, Invalid response, status code: 403 - Data: {"errorGroup":"FORBIDDEN","errorCode":"AUT_00055","errorMessage":"User does not have permission to access the requested resource"}
```

This makes the integration no longer usable with regular users and thus requires one to use an admin user for the alarm system AND requires you to disable 2FA for such a user...

To me personally, I value the security of my system more than the firmware update status. I'm not sure right now; if or how, to move forward on this.

Reply:
ah sorry, I actually have an admin user with MFA disabled, so I didn't spot it.

As  I can see, `vusre` is supporting MFA, so, would it be acceptable to require an admin user and support MFA through config_flow ?  (I think `synology_dsm` is already working the same way)